This patch removes anything relating to examples,
nightly-only features, windows-only dependencies.

diff --git a/Cargo.toml b/Cargo.toml
index 27dc196..e5b9d5b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,14 +24,6 @@ readme = "README.md"
 license = "MIT/Apache-2.0"
 repository = "https://github.com/rust-lang/backtrace-rs"
 
-[[example]]
-name = "backtrace"
-required-features = ["std"]
-
-[[example]]
-name = "raw"
-required-features = ["std"]
-
 [[test]]
 name = "skip_inner_frames"
 required-features = ["std"]
@@ -67,15 +59,6 @@ optional = true
 [dependencies.cfg-if]
 version = "0.1.6"
 
-[dependencies.compiler_builtins]
-version = "0.1.2"
-optional = true
-
-[dependencies.core]
-version = "1.0.0"
-optional = true
-package = "rustc-std-workspace-core"
-
 [dependencies.cpp_demangle]
 version = "0.2.3"
 optional = true
@@ -120,12 +103,8 @@ gimli-symbolize = ["addr2line", "findshlibs", "memmap", "goblin"]
 kernel32 = []
 libbacktrace = ["backtrace-sys"]
 libunwind = []
-rustc-dep-of-std = ["backtrace-sys/rustc-dep-of-std", "cfg-if/rustc-dep-of-std", "core", "compiler_builtins", "libc/rustc-dep-of-std", "rustc-demangle/rustc-dep-of-std"]
 serialize-rustc = ["rustc-serialize"]
 serialize-serde = ["serde"]
 std = []
 unix-backtrace = []
-verify-winapi = ["winapi/dbghelp", "winapi/handleapi", "winapi/libloaderapi", "winapi/minwindef", "winapi/processthreadsapi", "winapi/synchapi", "winapi/winbase", "winapi/winnt"]
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.3"
-optional = true
+
